exonum-light-client/README.md

```diff
@@ -121,7 +121,18 @@ if available, transaction location and result:
 Optional<TransactionResponse> response = exonumClient.getTransaction(txHash);
 ```
 * `txHash` is a hash of the transaction to search.
-  
+
+### Retrieving service info
+To build a transaction, service id is needed. It can be obtained with either
```

Comment:
This documentation seems out of place. I think it shall be included in the section on creating a transaction message, saying that the service id, required to create a message, can be obtained, if needed, by the service name:
```java
// Update the code to be correct and compile!
int serviceId =  exonumClient.getServiceInfoByName(serviceName)
    .mapToInt(#getId)
    .orElseThrow(() -> new IllegalStateException("No service with the given name found: " + serviceName);
```

This section may remain, but rewritten as just the way to get information on active services in the blockchain.

Reply:
> but rewritten as just *the way to get information on active services in the blockchain*.

It still starts as "To build a transaction, service id is needed."